metadata-editor/app/model/UsageRightsProperty.scala

```diff
@@ -19,22 +19,25 @@ case class UsageRightsProperty(
   examples: Option[String] = None
 )
 
-
 object UsageRightsProperty {
   type OptionsMap = Map[String, List[String]]
```

Comment:
It's not directly related to this PR, but I found this very difficult to read – not Scala's `Option`, but a small-O option for the `UsageRightsProperty`, related to – usage rights, but I'm still not sure what this is exactly! Doesn't help that I don't understand the domain too well, but a clearer name might make things easier. Happy to submit another PR if needed.